src/diffpy/srmise/applications/plot.py

```diff
@@ -504,7 +504,7 @@ def on_draw(event):
     invisiblelabel.set_visible(True)
     visiblelabel = labeldict[fig]
     bbox = invisiblelabel.get_window_extent(invisiblelabel._renderer)
-    bbox = bbox.inverse_transformed(ax_main.transAxes)
+    bbox = bbox.transformed(ax_main.transAxes.inverted())
```

Comment:
`inverse_transformed` is depreciated in `matplotlib. 

Ref: https://matplotlib.org/3.3.0/api/prev_api_changes/api_changes_3.3.0/deprecations.html#bboxbase-inverse-transformed